README.md

```diff
@@ -41,6 +41,17 @@ project `<aosp>/external/sl4a`
 It can be built like a regular system app with `mm` commands. It needs to be
 signed with the build you use on your Android devices.
 
+__Installation__
+
+For Python3
+```
+sudo python3 setup.py install
+```
+For Python2
+```
+sudo python setup.py install
```

Comment:
Do we need to add a clarification in the 'execute' instructions below this to choose python3 or python depending on how it was installed?

Currently the 'execute' steps just say 'python'.